independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/injection/assignability/generics/AssignabilityWithGenericsTest.java

```diff
@@ -29,7 +32,11 @@
             StringListConsumer.class, ListConsumer.class, ProducerBean.class, DefinitelyNotBar.class,
             Bar.class, GenericInterface.class, AlmostCompleteBean.class, ActualBean.class,
             BetaFace.class, GammaFace.class, GammaImpl.class, AbstractAlpha.class, AlphaImpl.class,
-            BeanInjectingActualType.class, FooTyped.class);
+            BeanInjectingActualType.class, FooTyped.class,
+            ScoreManager1.class, MyScore1.class, ScoreManagerBeanProvider1.class,
+            ScoreManager2.class, Score2.class, MyScore2.class, ScoreManagerBeanProvider2.class
+//            , ScoreManager3.class, Score3.class, MyScore3.class, ScoreManagerBeanProvider3.class
```

Comment:
Uncomment this line and the lines at the bottom of this file and it will fail at runtime bootstrap (killing the entire test).